webapp/app.py

```diff
@@ -219,6 +219,15 @@ def deleted_error(error):
     return flask.render_template("410.html", message=error.description), 410
 
 
+@app.errorhandler(429)
+def too_many_requests(error):
+    """
+    Endpoint abuse error
+    """
+    custom_error = f'{error.description}. Please try again tomorrow.'
```

Comment:
In QAing I realised this message won't work any more. I got "The rate limit is: 3/minute. Please try again tomorrow.". It'll have to be "Please try again later."
```suggestion
    custom_error = f'{error.description}. Please try again later.'
```